modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -264,11 +264,8 @@ public ClusterManagementGroupManager(
      * @param cmgNodeNames Names of nodes that will host the Cluster Management Group.
      * @param clusterName Human-readable name of the cluster.
      */
-    public void initCluster(
-            Collection<String> metaStorageNodeNames,
-            Collection<String> cmgNodeNames,
-            String clusterName
-    ) throws NodeStoppingException {
+    public void initCluster(Collection<String> metaStorageNodeNames, Collection<String> cmgNodeNames, String clusterName)
```

Comment:
This formatting looked fine, what was the problem?

Reply:
Parameters fit one line, so I reformatted it. It seems ok to wrap throws declarations to next line

Reply:
"if it fits, it sits". Are rules are not **that** strict, I'm not a big fan of reformatting a code that has nothing to do with your changes

Reply:
Reverted